tests/new-range.spec.tsx

```
@@ -43,6 +43,89 @@ describe('NewPicker.Range', () => {
+
+  beforeEach(() => {
+    rangeRect = {
+      x: 0,
+      y: 0,
+      width: 200,
+      height: 100,
+    };
+
+    document.documentElement.scrollLeft = 0;
+    // jest.useFakeTimers();
+  });
```

Biome

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)